python/friesian/src/bigdl/friesian/feature/utils.py

```diff
@@ -160,13 +162,29 @@ def encode_target_(tbl, targets, target_cols=None, drop_cat=True, drop_fold=True
                     new_out_target_mean[out_col] = target_mean
             out_target_mean = new_out_target_mean
 
+        all_size = target_code.size()
+        limit_size = 1000000
```

Comment:
how this limit size is determined?

Reply:
most of the times, size of each categorical feature stay within 1m.